samples/word/01-basics/insert-and-get-pictures.yaml

```diff
@@ -1,4 +1,4 @@
-id: word-basics-insert-and-get-pictures
+id: word-basics-insert-and-get-pictures-
```

Comment:
is it actually my tool that adds all these dashes?  if so, don't worry about fixing them, I'll fix it via the tool itself.